osu/clients.py

```diff
@@ -49,8 +52,20 @@ async def osz_files_from_beatmapset_ids(cls, beatmapset_ids: List[int], no_video
 
         return osz_files
 
+class DbxClient:
+    client = None
+    dbx_path = "/mappapakka/"
+
+    @classmethod
```

Comment:
for the time being it might be better to keep this as a class, since behaviour doesn't really change between either (and stops the interpreter from yelling at me it could be made static). And you can make an argument that its difficult to convert this from static to class later on if we need to access some variables (highly doubt)